nebari/cli/keycloak.py

```diff
@@ -50,3 +51,26 @@ def list_users(
     args = ["listusers"]
 
     do_keycloak(config_filename, *args)
+
+
+@app_keycloak.command(name="export-users")
+def export_users(
+    config_filename: str = typer.Option(
+        ...,
+        "-c",
+        "--config",
+        help="nebari configuration file path",
+    ),
+    realm: str = typer.Option(
+        "nebari",
+        "--realm",
+        help="realm from which users are to be exported",
+    ),
+):
+    """Export the users in Keycloak."""
+    if isinstance(config_filename, str):
+        config_filename = Path(config_filename)
+
+    r = export_keycloak_users(config_filename, realm=realm)
+
+    print(json.dumps(r, indent=4))
```

Comment:
Looks great and its working as expected. Can we have `-o` output flag to save as a json file as well (not print when flag passed, could be used on CI in the future )

Reply:
Personally I don't think we need this feature. Can't we just run the command and redirect the output? e.g. `command > output.json`

Reply:
Indeed I forgot that was possible. Then its resdy to go.